spec/concepts/webhook_delivery/operation/create_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+require 'webmock_helper'
+require 'support/shared_contexts/workspace_context'
+require 'support/shared_examples/operation'
+
+describe WebhookDelivery::Create do
+  describe '.run' do
+    include_context 'workspace context'
+
+    let(:webhook) do
+      Webhook::Create.(webhook: { name: 'Example webhook',
+                                  url: 'http://example.io/webhooks',
+                                  workspace_id: workspace.id }).model
+    end
+
+    let(:params) { { webhook_delivery: { webhook_id: webhook.id } } }
+
+    subject(:operation) { WebhookDelivery::Create.run(params).last }
+
+    before { stub_request(:post, webhook.url).to_return(status: 200) }
+
+    include_examples 'create operation', WebhookDelivery
+
+    describe 'request headers' do
+      subject(:headers) { operation.model.request['headers'] }
+
+      it { is_expected.to include('Content-Type' => 'application/json') }
+      it { is_expected.to include('User-Agent' => 'Emilito-Hookshot') }
+      it { is_expected.to include('X-Emilito-Event') }
+      it { is_expected.to include('X-Emilito-Delivery' => operation.model.id) }
```

Comment:
Rename test case to: `it "set Webhook::Delivery uuid as X-Emilito-Delivery header"`